src/walletSdk/Types/sep38.ts

```diff
@@ -0,0 +1,77 @@
+import { AxiosInstance } from "axios";
+import { Anchor } from "../Anchor";
+import { AuthToken } from "./auth";
+
+export interface Sep38Info {
+  assets: Array<Sep38AssetInfo>;
+}
+
+export interface Sep38AssetInfo {
+  asset: string;
+  sell_delivery_methods?: Array<Sep38DeliveryMethod>;
+  buy_delivery_methods?: Array<Sep38DeliveryMethod>;
+  country_codes?: Array<string>;
+}
+
+export interface Sep38DeliveryMethod {
+  name: string;
+  description: string;
+}
+
+export type Sep38Params = {
+  anchor: Anchor;
+  httpClient: AxiosInstance;
+  authToken?: AuthToken;
+};
+
+export interface Sep38PricesParams {
+  sellAsset: string;
+  sellAmount: string;
+  sellDeliveryMethod?: string;
+  buyDeliveryMethod?: string;
+  countryCode?: string;
+}
+
+export interface Sep38PricesResponse {
+  buy_assets: Array<Sep38BuyAsset>;
+}
+
+export interface Sep38BuyAsset {
+  asset: string;
+  price: string;
+  decimals: number;
+}
+
+export interface Sep38PriceParams {
```

Comment:
I think we should have 2 different interfaces, 1 with mandatory `sellAmount`, the other with mandatory `buyAmount` or validate that one of sell/buy amount is set in the request method

Reply:
good point. I like the validate way, it's similar to how we're handling elsewhere in the sdk

https://github.com/stellar/typescript-wallet-sdk/pull/84/commits/28f2ef03c8a74aaa4e2f10c349c3f1138b2b7837